k8s/pkg/apis/networkservice/v1alpha1/types.go

```diff
@@ -61,6 +61,7 @@ type NetworkServiceEndpoint struct {
 
 type NetworkServiceEndpointSpec struct {
 	NetworkServiceName string `json:"networkservicename"`
+	Payload            string `json:"payload"`
```

Comment:
Why do we need `Payload` here? As far as I know we already have one in `NetworkService.Spec`

Reply:
There can be an issue when the `NSE` resource is in registry, but the corresponding `networkservice` resource has been deleted.